numpy/core/src/umath/test_rational.c.src

```diff
@@ -399,8 +399,7 @@ pyrational_new(PyTypeObject* type, PyObject* args, PyObject* kwds) {
                 "expected rational or numerator and optional denominator");
         return 0;
     }
-    x[0] = PyTuple_GET_ITEM(args,0);
-    x[1] = PyTuple_GET_ITEM(args,1);
+    x[0] = PyTuple_GET_ITEM(args, 0);
```

Comment:
Isn't this still wrong when `size == 0`?

Reply:
Heh, yeah, dunno how I could have missed that check was missing.

Reply:
I guess its just testing code, but frankly, eying it shortly, there is likely more messed up here.